bin/templates/cordova/version

```diff
@@ -19,16 +19,5 @@
     under the License.
 */
 
-let VERSION = 'version undefined';
```

Comment:
The moved method does not return this in case it cannot get these files

Reply:
This is not needed anymore. It will always return something now.

Reply:
Ok makes sense